apps/trading/pages/markets/trade-grid.tsx

```diff
@@ -221,95 +221,106 @@ export const TradeMarketHeader = ({
+        <ResizableGridPanel
+          priority={LayoutPriority.High}
+          minSize={200}
+          preferredSize="50%"
+        >
+          <TradeGridChild>
+            <Tabs>
+              <Tab id="candles" name={t('Candles')}>
+                <TradingViews.Candles marketId={marketId} />
+              </Tab>
+              <Tab id="depth" name={t('Depth')}>
+                <TradingViews.Depth marketId={marketId} />
+              </Tab>
+            </Tabs>
+          </TradeGridChild>
         </ResizableGridPanel>
         <ResizableGridPanel
           priority={LayoutPriority.Low}
-          preferredSize="25%"
-          minSize={50}
+          preferredSize={330}
+          minSize={300}
         >
           <TradeGridChild>
             <Tabs>
-              <Tab id="positions" name={t('Positions')}>
-                <TradingViews.Positions />
+              <Tab id="ticket" name={t('Ticket')}>
+                <TradingViews.Ticket marketId={marketId} />
               </Tab>
-              <Tab id="orders" name={t('Orders')}>
-                <TradingViews.Orders />
+              <Tab id="info" name={t('Info')}>
+                <TradingViews.Info
+                  marketId={marketId}
+                  onSelect={(id: string) => {
+                    onSelect(id);
+                  }}
+                />
               </Tab>
-              <Tab id="fills" name={t('Fills')}>
-                <TradingViews.Fills />
+            </Tabs>
+          </TradeGridChild>
+        </ResizableGridPanel>
+        <ResizableGridPanel
+          priority={LayoutPriority.Low}
+          preferredSize={430}
+          minSize={200}
+        >
+          <TradeGridChild>
+            <Tabs>
+              <Tab id="orderbook" name={t('Orderbook')}>
+                <TradingViews.Orderbook marketId={marketId} />
               </Tab>
-              <Tab id="accounts" name={t('Collateral')}>
-                <TradingViews.Collateral />
+              <Tab id="trades" name={t('Trades')}>
+                <TradingViews.Trades marketId={marketId} />
               </Tab>
             </Tabs>
           </TradeGridChild>
         </ResizableGridPanel>
       </ResizableGrid>
+    </ResizableGridPanel>
+    <ResizableGridPanel
+      priority={LayoutPriority.Low}
+      preferredSize="25%"
+      minSize={50}
+    >
+      <TradeGridChild>
+        <Tabs>
+          <Tab id="positions" name={t('Positions')}>
+            <TradingViews.Positions />
+          </Tab>
+          <Tab id="orders" name={t('Orders')}>
+            <TradingViews.Orders />
+          </Tab>
+          <Tab id="fills" name={t('Fills')}>
+            <TradingViews.Fills />
+          </Tab>
+          <Tab id="accounts" name={t('Collateral')}>
+            <TradingViews.Collateral />
+          </Tab>
+        </Tabs>
+      </TradeGridChild>
+    </ResizableGridPanel>
+  </ResizableGrid>
+);
+const MainGridWrapped = memo(MainGrid);
```

Comment:
question (non blocking): Was there a specific reason for memo here? Should we be wrapping TradePanels with memo too? It also renders the header.

Reply:
It was added for avoid re-rendering all child components, what was caused by passing marketID down to the children. But now I'm gonna change a little bit the conception, following Bart's advice it can be better to break down the update to all children separately

Reply:
So TradePanel also is wrapped with `memo`. It just prevent re-render by restictly checking the props. Without it re-render happens also when `market` object changes in the parent space, what occur quite often